### PR TITLE
Electric Coin Company

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by ebfull** [comment]
I don't know how this works legally and I wish we didn't put these silly preambles into our code. (I think once you start putting them you have to continue for some ridiculous reason.)

**Comment by rex4539** [comment]
Ping

File: LICENSE-MIT [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Zcash Company
+Copyright (c) 2019 Electric Coin Company
```
Comment (daira) [inline LICENSE-MIT]: It should be 2017-2019.

**Comment by daira** [comment]
@ebfull: these particular preambles are in the `LICENSE-MIT` files for subprojects, not the code. I totally agree with you about code preambles. As far as I know, it's possible to strip those out as long as the actual license/copyright doesn't change, but we should discuss that on #2887.

**Pull request merged**